README.md

```diff
@@ -22,7 +22,6 @@ Pre-requisite
 
 * knowledge of GitHub, not a must though you can learn basics in a couple of hours.
 * Presence on Telegram Group (install telegram first if an error or join from the desktop )
-* Fill this form with SOP(statement of purpose)
```

Comment:
if you have it why don't you add the form link. @Shivank98 I recollect you saying this was a requirement?

Reply:
Yeah, it was but now we can do it like any other Open Source project.

Reply:
looks go to me.

Reply:
@theSage21 , please merge this one too.